dashboards/flow-data-per-organization.json

```diff
@@ -19,7 +19,7 @@
   "links": [],
   "panels": [
     {
-      "content": "<div style=\"float:right;\">\n<a class=\"btn navbar-button gf-timepicker-nav-btn\" href=\"/grafana/d/-l3_u8nWk/individual-flows?var-src=$Organization\" style=\"text-decoration: none; color: #52545C; font-weight: bold;\">See Individual Flows for $Organization</a>\n</div>\n<br clear=\"both\" />\n<div style=\"padding-left: 10%;\">\n<center><h1><b>Flow Data by Organization</b></h1></center>\n<center>Select an Organization from the list above.</center>\n<center>Please note that the dropdown will only show 1000 results at a time, so you may need to filter (start typing) to find a specific organization.</center>\n<center>This dashboard shows top flows with the specified organization as the source or destination in a given time period.</center>\n</div>\n\n\n\n<br>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
+      "content": "<div style=\"float:right;\">\n<a class=\"btn navbar-button gf-timepicker-nav-btn\" href=\"/grafana/d/-l3_u8nWk/individual-flows?var-src=$Organization\" style=\"position-fixed; left: -100px; width: 300px; text-decoration: none; color: #52545C; font-weight: bold;\">See Individual Flows for $Organization</a>\n</div>\n<br clear=\"both\" />\n<div style=\"padding-left: 10%;\">\n<center><h1><b>Flow Data by Organization</b></h1></center>\n<center>Select an Organization from the list above.</center>\n<center>Please note that the dropdown will only show 1000 results at a time, so you may need to filter (start typing) to find a specific organization.</center>\n<center>This dashboard shows top flows with the specified organization as the sender or receiver in a given time period.</center><center>All times are displayed in browser local time.</center>\n</div>\n\n\n\n<br>\n<!-- Global site tag (gtag.js) - Google Analytics -->\n<script async src=\"https://www.googletagmanager.com/gtag/js?id=UA-142763676-1\"></script>\n<script>\n  window.dataLayer = window.dataLayer || [];\n  function gtag(){dataLayer.push(arguments);}\n  gtag('js', new Date());\n\n  gtag('config', 'UA-142763676-1');\n</script>",
```

Comment:
Same as above, also squashed some of the changes I had made.

Reply:
Fixed